plugin-server/src/main/services/mmdb.ts

```diff
@@ -61,6 +61,7 @@ async function decompressAndOpenMmdb(brotliContents: Buffer, filename: string):
 async function fetchAndInsertFreshMmdb(hub: Hub): Promise<ReaderModel> {
     const { db } = hub
 
+    // TODO: use local GeoLite2 on container at share/GeoLite2-City.mmdb instead of downloading it each time
```

Comment:
clever!

cc: @Twixes , I remember you talking about a reason why we are doing things like so, remind me?

Reply:
I'm not Twixes but I think it was due to what I've mentioned at https://github.com/PostHog/posthog-cloud/pull/206#issuecomment-1189978585.

Reply:
Our license prohibits us from distributing this `.mmdb` file in any publicly accessible repository -> we can't make it downloadable to others. Bundled into a docker container at run time is probably fine, except for the increase in size, and the eventual staleness of the data.

Reply:
In this case we need to remove `.mmdb` from this PR and have a script that will grab it for development (since it won't be in the repo by default)

Reply:
Accordingly to @charlescook-ph [it shouldn't be an issue to package it](https://github.com/PostHog/posthog-cloud/pull/206#issuecomment-1190257758)

Reply:
Yes, the current setup optimizes for ease of local development and caution license-wise. Packaging with the Docker image seems pretty fine though actually.